geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -322,6 +322,9 @@ pub enum SlotStatus {
 
     /// First Shred Received
     FirstShredReceived,
+
+    /// Completed
```

Comment:
Might be good to clarify what completed means here. I believe it means all shreds have been received, block has been replayed & frozen